example/src/modules/Story/StoryScreen.tsx

```diff
@@ -74,6 +81,18 @@ const StoryScreen = () => {
             )}
             //Callback when status view completes
             onComplete={closeStory}
+            overlayViewPostion={'top'}
+            renderOverlayView={(item: StoryType) => (
+              <TouchableOpacity
```

Comment:
@nilesh-simform Instead of defining this multiple times, it is better to create a component and re-use it.